.github/workflows/upload-cloud-storage-it.yml

```diff
@@ -0,0 +1,93 @@
+name: upload-cloud-storage Integration
+on: [push]
```

Comment:
Please update to only run when files in the appropriate subdirectory change (see others for an example)

Reply:
done